lib/src/main/java/io/ably/lib/types/ClientOptions.java

```diff
@@ -195,4 +196,10 @@ public ClientOptions(String key) throws AblyException {
      * before responding.
      */
     public boolean pushFullWait = false;
+
+    /**
+     * Allows provide custom Local Device storage. In a case nothing is provided default implementation
+     * using SharedPreferences is used.
+     */
+    public Storage storage = null;
```

Comment:
I think we could perhaps rename this field to `localStorage` (the interface name can remain as `Storage`).

Reply:
https://github.com/ably/ably-java/pull/681/commits/16bf5233f352b5817f5afae1af1e95ab7e283c7f